prow/config.yaml

```diff
@@ -1655,29 +1655,58 @@ presubmits:
   - name: pull-tekton-triggers-integration-tests
     labels:
       preset-presubmit-sh: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      pipeline-kind-e2e: "true"
```

Comment:
We should probably rename this label, since its purpose is to ensure that kind e2e jobs each end up on their own node, regardless of project. But that doesn't need to be done here. =)

Reply:
I'll do that as part of https://github.com/tektoncd/plumbing/issues/1281